src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -171,6 +171,8 @@ public String deploy(AppDeploymentRequest request) {
 				.then(() -> operations.applications()
 					.start(StartApplicationRequest.builder()
 						.name(name)
+						.stagingTimeout(deploymentProperties.getStagingTimeout())
```

Comment:
When we deploy using the deployer, we first set no-start and *then* start, hence those timeouts only make sense on the start() call.